workers/share/scripts/deploy-dev.sh

```
@@ -0,0 +1,78 @@
+  printf '%s' "$value"
+}
+
+put_worker_secret() {
+  local key="$1"
+  local value="$2"
+  printf '%s' "$value" | bunx wrangler secret put "$key" --config wrangler.dev.toml --name "$name" >/dev/null
+}
+
+raw="${1:-${CMUX_SHARE_DEV_SLUG:-$(git config user.email 2>/dev/null | cut -d@ -f1 || true)}}"
+raw="${raw:-${USER:-}}"
+slug="$(printf '%s' "$raw" | tr 'A-Z' 'a-z' | tr -c 'a-z0-9-' '-' | sed 's/--*/-/g; s/^-//; s/-*$//')"
```

Shellcheck (0.11.0)

[info] Use '[:upper:]' to support accents and foreign alphabets.

(SC2019)